lib/entangler/executor/background/base.rb

```diff
@@ -42,12 +42,12 @@ def kill_off_threads
         private
 
         def listener
-          @listener ||= begin
-            Listen::Listener.new(base_dir, ignore!: @opts[:ignore]) do |modified, added, removed|
-              process_local_changes(generate_entangled_files(added, :create) +
-                                        generate_entangled_files(modified, :update) +
-                                        generate_entangled_files(removed, :delete))
-            end
+          @listener ||= Listen::Listener.new(base_dir,
+                                             ignore!: @opts[:ignore],
+                                             force_polling: @opts[:force_polling]) do |modified, added, removed|
```

Comment:
It's fine to always provide `force_polling` since it's [only enabled if truthy](https://github.com/guard/listen/blob/fd85e1cb2375767e3cbc4b5743ff50061e8a6c75/lib/listen/adapter.rb#L19).